packages/video_player/video_player/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.2.11
+
+* Fix persisting of hasError even after successful initialize
```

Comment:
Nit: Fixes
Also needs a period at the end.

(See changelog styleguide linked from the checklist)